### DISCUSSION TITLE
Adding Matter.js APIs to ECMA-419

### DISCUSSION THREAD

**Discussion by HipsterBrown** [matter-js/matter.js#692] [General]
Hello!

I am a member of [TC53](https://ecma-international.org/technical-committees/tc53/), the technical committee responsible for the [ECMA-419](https://ecma-international.org/publications-and-standards/standards/ecma-419/) standard of JavaScript APIs for embedded systems. We are currently deciding on topics for the 3rd edition of the spec and have started exploring how Matter APIs could be integrated. 

Given the excellent work in this project, the committee wants to invite the maintainers to join a meeting to discuss the topic of Matter within the standard. If any of the maintainers are interested, we can work out a time during our normal meeting dates or a separate event if needed.

**Comment by Apollon77** [comment]
Hello @HipsterBrown, thank you for your consideration and contact. We would be happy to join a meeting. I am based in Germany. The second main developer @lauckhart is based in US. If you like just contact me at matter@fischer-ka.de or on discord (apollon77) to sync on the details.
